tests/auto/gltfexporter/gltfexporter.pro

```diff
@@ -0,0 +1,39 @@
+# gltfparser.pro
+#
+# This file is part of Kuesa.
+#
+# Copyright (C) 2018 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+# Author: Mike Krus <mike.krus@kdab.com>
```

Comment:
I don't think Mike actually wrote this :)

Reply:
likewise, in the latest commit it has already been fixed (https://github.com/KDAB/kuesa/pull/16/files#diff-1f8bf50859e4b422c66a957bfb568cafR6)